### ISSUE TITLE
Importing Images into SketchUp

### ISSUE THREAD

**Issue by Ellis307** [SASDigitalHumanitiesTraining/3D#21]
(Sorry me again)

I am about to have a stab at the village garden 3.2 and I was wondering…is there a way to import the image into Sketchup and scale it according to the scale given in the image to save time flicking backwards and forwards?

Ellis

**Comment by Ellis307** [comment]
The issue with the image is that there is a scale but no overall size for the structure. I guess its possible to keep stretching the image until I measure the scale at the side of the image to be 10m. Its this one 
![](https://static.cambridge.org/binary/version/id/urn:cambridge.org:id:binary:20171205070135153-0150:9781139033022:fig3_2.png)

**Comment by Ellis307** [comment]
![](https://static.cambridge.org/binary/version/id/urn:cambridge.org:id:binary:20171205070135153-0150:9781139033022:fig3_2.png)

**Comment by gabrielbodard** [comment]
Yes, I did a very similar thing: used the scale on the image to calculate the size in metres of the whole building (with a ruler against my computer screen!), and drew a rectangle with those dimensions, then pasted the image as a texture in that rectangle. It's a bit crude either way…

**Comment by oshepard** [comment]
@gabrielbodard and @Ellis307 

The easiest way is editing the picture previously with another software so it has the size you want (eg. Photoshop or even Paint). They have "rulers" inside and you can adjust it as you prefer.

**Comment by Ellis307** [comment]
Gotcha! That's really helpful thank you.
